README.md

```diff
@@ -265,7 +263,7 @@ export MASTER_KEY=$(openssl rand -hex 32)
       --set artifactory.license.dataKey=artifactory.cluster.license \
       --set artifactory.metrics.enabled=true \
       -n $INST_NAMESPACE
-   ```
+   ```s
```

Comment:
Is this `s` intentional?

Reply:
Yes. The customer can provide their own license manually. I had issues with customers not provided the license secret, and it made helm deployment to fail.

Reply:
Ohh wait I missed the `s` - fixing it now